app/src/main/java/org/sopt/dosopttemplate/presentation/auth/login/LoginViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package org.sopt.dosopttemplate.presentation.auth.login
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+import org.sopt.dosopttemplate.server.ServicePool.authService
+import org.sopt.dosopttemplate.server.auth.request.RequestLoginDto
+import org.sopt.dosopttemplate.server.auth.response.ResponseLoginDto
+
+class LoginViewModel : ViewModel() {
+    private val _loginResult: MutableLiveData<ResponseLoginDto> = MutableLiveData()
+    val loginResult: MutableLiveData<ResponseLoginDto> = _loginResult
+
+    private val _loginSuccess: MutableLiveData<Boolean> = MutableLiveData()
+    val loginSuccess: MutableLiveData<Boolean> = _loginSuccess
+
+    fun login(id: String, password: String) {
+        viewModelScope.launch {
+            kotlin.runCatching {
+                authService.login(RequestLoginDto(id, password))
+            }.onSuccess {
+                if (it.isSuccessful) {
+                    loginResult.value = it.body()
```

Comment:
```suggestion
_loginResult.value = it.body()
```
수정은 뷰모델 내에서만 할 수 있도록 private 변수를 써주세용

Reply:
반영했어요!